best-practices.md

```diff
@@ -52,11 +46,23 @@ All rules below should be followed by contributors to this repo. Contributors sh
   - To disambiguate a task or workflow file from it's contents, you can respectively add the `_tasks` or `_wf` suffix in the import section
 - Whenever possible, prefer a Docker image maintained by an external source (such as BioContainers) rather than creating your own image
 - When adding a Dockerfile to this repository, follow the below conventions
-  - The `Dockerfile` should be nested under the `docker/` directory, a folder with a name for the image (in most cases the name of the primary tool), and finally a folder named after the version being built.
+  - Create a directory under the `docker/` directory and choose an appropriate name (likely shared with the underlying tool). The `Dockerfile` should be nested under this new directory. Then create a `package.json` alongside the `Dockerfile`. The `package.json` file is required to contain two JSON fields (`name` and `version`). It can optionally contain a `revision` field.
   - Docker images should be versioned according to the following convention
-    - Start with the version of whatever tool is named in the path to the `Dockerfile`
+    - The `version` should be shared with whatever underlying tool is being used
       - If no specific tool is named (e.g. the `util` image), default to SemVer. Ignore the next 3 bullet points.
-    - Followed by a dash-zero (`-0`)
-      - If the Docker image gets updated, *without* updating the base tool's version, increment the number after the dash (`-`) by one
-      - If the Docker image gets updated, *including* updating the base tool's version, revert back to a dash-zero (`-0`)
+    - The revision should start with zero (`0`)
+      - If the Docker image gets updated, *without* updating the base tool's version, increment the number by one
+      - If the Docker image gets updated, *including* updating the base tool's version, revert back to zero
 - general purpose tasks can use the `util` image maintained in this repo
+- The `description` should be in active voice, beginning the first sentence with a verb
+  - Each task/workflow is _doing_ something. The first sentence should be a succinct description of what that "something" is.
```

Comment:
```suggestion
  - Each task/workflow is _doing_ something. The first sentence should be a succinct description of what that "something" is.
  - The `description` key should be succinct. Generally, one sentence about 100 characters long is appropriate.
```
note: I'm still playing with how long the strings rendered as descriptions should be on the docs site. 100 is approximate at this point in time.